AddTransaction.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { describe, it, expect } from 'vitest'
+import { mount } from '@vue/test-utils'
+import AddTransaction from '@/components/AddTransaction.vue'
+
+describe('AddTransaction elements', () => {
+  const AddTransactionWrapper = mount(AddTransaction)
+
+  it('should render correctly', () => {
+    expect(AddTransactionWrapper.exists()).toBe(true);
+  })
+
+  it('should have a button to submit a new transaction data', () => {
+    const button = AddTransactionWrapper.find('button')
```

Comment:
```suggestion
    const addTransactionButton = AddTransactionWrapper.find('button') || const addTransactionButton = wrapper.find('button')
```
The button you want to test must be meaningful.